source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/HelperFragment.java

```diff
@@ -51,6 +51,8 @@
     static final int RC_SIMPLE_UI = 9003;
     static final int RC_SELECT_SNAPSHOT_UI = 9004;
     static final int RC_CAPTURE_OVERLAY_UI = 9005;
+    static final int RC_SELECT_OPPONENTS_UI = 9006;
+    static final int RC_SHOW_INBOX_UI = 9007;
```

Comment:
RC_SHOW_INBOX_UI->RC_INBOX_UI

Reply:
Now everything has this format:
get[name]Intent       to get intent from Java SDK
RC_[name]_UI        field in HelperFragment
show[name]_Ui       method in HelperFragment
[name]UiRequest    file for request.